seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/VariablesSubstitute.java

```diff
@@ -25,25 +25,23 @@
 import java.util.Map;
 import java.util.UUID;
 
-public final class StringTemplate {
```

Comment:
I don't suggest to change Class name, seatuunel-apis depend on it, so we should keep it stable and compatible.

Reply:
make sense